go.sum

```diff
@@ -1484,3 +1484,5 @@ sigs.k8s.io/yaml v1.2.0 h1:kr/MCeFWJWTwyaHoR9c8EjH9OumOmoF9YGiZd7lFm/Q=
 sigs.k8s.io/yaml v1.2.0/go.mod h1:yfXDCHCao9+ENCvLSE62v9VSji2MKu5jeNfTrofGhJc=
 vitess.io/vitess v0.10.3-0.20220912060457-89093912dab0 h1:h5gorSwS7aVtFd4sWwQjuC4+mbxbIuS/ryiZABU3zR4=
 vitess.io/vitess v0.10.3-0.20220912060457-89093912dab0/go.mod h1:D2+xmC1AhIMGqR6Y3Qf18ribmrAotiTNuLXkZ3cT++E=
```

Comment:
These lines should also be removed. We should also run `go mod tidy` in the workflow